integration/kubernetes/helm-chart/alluxio/templates/worker/daemonset.yaml

```diff
@@ -204,6 +231,9 @@ spec:
             - name: conf-volume
               mountPath: /opt/alluxio/conf
               readOnly: false
+            - name: logs-volume
+              mountPath: /opt/alluxio/logs
+              readOnly: false
```

Comment:
See comment for Alluxio Master, but could you make this a configurable value in `values.yaml`? `.Values.worker.volumes.log.path` or something like that, and replace the usage of `/opt/alluxio/logs` in this file.

Reply:
updated to use `.Values.hub.volumes.log.path` since it's a hub-only volume